features/package_development_flow/build.bat

```diff
@@ -0,0 +1,3 @@
+@ECHO ON
+
+python demo.py
```

Comment:
If we can expand here the command it will be better, like in the rest of the examples. People will reach a well know `.sh` or `.bat` and copy&paste command from it.

Reply:
done!